report_templates/host_-_compare_content_hosts_packages.erb

```diff
@@ -1,5 +1,5 @@
 <%#
-name: Host - compare content hosts packages
+name: Host - Compare content hosts packages
```

Comment:
Isn't this cause troubles? Will we need migration just because of this?

Reply:
Oh it's merged recently?

Reply:
Good catch, I did not mean to change the name.